core/src/test/java/org/testcontainers/junit/GenericContainerRuleTest.java

```diff
@@ -124,6 +126,13 @@ public static void setupContent() throws FileNotFoundException {
             .withExtraHost("somehost", "192.168.1.10")
             .withCommand("/bin/sh", "-c", "while true; do cat /etc/hosts | nc -l -p 80; done");
 
+    /**
+     * Redis with shared memory
+     */
+    @ClassRule
```

Comment:
I think using the `@ClassRule` for all containers in this test class is quite bad and we should instead start the containers in the test itself. It's not your fault of course and should not be part of this PR, just a heads up for the others.

Reply:
Moved to a try-with-resources ; also made it busybox to lighten the container that starts up.